site/_posts/2022-04-14-announcing-fusionauth-1.36.md

```diff
@@ -0,0 +1,66 @@
+
+This feature is particularly helpful in cases where you want to use a third-party SCIM-compliant backend to provision users and keep user data in sync. SCIM helps you manage users across systems like GitHub, Salesforce, and Trello, among others.
+
+SCIM data formats vary wildly so a new lambda is available. This lambda will take care of mapping from the SCIM provided user data into the FusionAuth user data schema, allowing for integration flexibility.
+
+This functionality is only available for our Enterprise users; learn more by visiting [the pricing page](/pricing) or [contacting our sales team](/contact).
+
+{% include _image.liquid src="/assets/img/blogs/release-1-36/scim-tenant-settings.png" alt="SCIM tenant settings" class="img-fluid" figure=true %}
+
+## Nintendo Online Identity Provider
+
+Nintendo Online has been added as an identity provider, allowing users to log in using their Nintendo Online accounts.
+
+{% include _image.liquid src="/assets/img/blogs/release-1-36/nintendo-login.png" alt="Nintendo added as Idp." class="img-fluid" figure=true %}
+
+This feature is available to our Enterprise and Essentials license holders as part of the “Advanced and gaming Idp” suite. Learn more about these editions by visiting [the pricing page](/pricing) or [contacting our sales team](/contact).
+ 
+## Webhook for user Link and Unlink
+
+A new webhook event is available for developers to trigger an action when a user links or unlinks to an identity provider.
+
+* This will allow developers to write a subsystem that immediately notices any new Idp accounts their users associate with their FusionAuth account (e.g. Xbox, Nintendo, PlayStation) and when these accounts are deleted.
+* Another system listens for those events and handles them: for instance it records the data for analytics, or adjusts functionality which is only available if you have an account at a certain IdP, like a special in-game item is only available to Xbox users or Nintendo users.
+
+Documentation for this web hook will be available [here](/docs/v1/tech/apis/webhooks#overview) soon.
+ 
+## The rest of it
+
+There were 17 issues, enhancements, and bug fixes included in this release. A selection of these include:
+
+* Custom password plugins that rely on dependencies in jarfiles are now more easily loaded. Required jars can now be  placed in the same directory as custom plugin, and will be loaded in a separate class loader.
+* OAuth2 Password Grant correctly returns MFA method
+* Support for higher volumes of logins when using a connector by making more conservative use of system resources
+Read more about all the changes in the [release notes](/docs/v1/tech/release-notes#version-1-36-1).
```

Comment:
I think 'Read more' needs a space, otherwise it gets pulled up into the list.

Reply:
Done